questions.txt

```diff
@@ -19,3 +19,78 @@ correct: $C\mathrm{e}^t$
 incorrect: $C\sin{t}$
 incorrect: $C\cos{t}$
 incorrect: $C\log{t}$
+
+------------------ Question
+
+type: TF
+prompt: The matrix, 
+<$A = \begin{bmatrix}
+<1 & 2 & 3\\
+<2 & 9 & 2\\
+<4 & 8 & 12\end{bmatrix}$, has rank 2.
+answer: true
+
+------------------ Question
+
+type: ESS
+prompt: Explain why the matrix, 
+\$A = \begin{bmatrix}
+\1 & 2 & 3\\
+\2 & 9 & 2\\
+\4 & 8 & 12\end{bmatrix}$, will have an eigenvalue equal to 0.
+example: exceptionally well!
+
+------------------ Question
+
+type: ORD
+prompt: $X$ follows a continuous distribution and has probability density
+\function
+>\[\begin{array}{l}f_X(x) = \frac{1}{2}x, \hspace{10pt} 0 \leq X \leq 2\\
+>\hspace{30pt}= 0, \hspace{10pt} \mathrm{elsewhere.}\end{array}\]
+>Order the following probability measures with the largest at the top.
+answer: The mode of $X$
+answer: The median of $X$
+answer: The mean of $X$
+
+------------------ Question
+
+type: MAT
+prompt: Match the letters to their position in the alphabet
+match_a: a
+match_b: 1
+match_a: b
+match_b: 2
+match_a: c
+match_b: 3
+
+------------------ Question
+
+type: FIL
+prompt: Upload a file please
+
+------------------ Question
+
+type: NUM
+prompt: Out of 10 how good is this question
+answer: 10
+tolerance: 1
+
+------------------ Question
+
+type: SR
+prompt: In a few words, how is this going
+example: very well
+
+------------------ Question
+
+type: OP
+prompt: How good is this going so far?
```

Comment:
Is there a blackboard doc giving a good explanation of what these question types mean or are for? Can we link to that?

Reply:
yes its at the top of the file:

> Each class corresponds to one of Blackboard's question types, for example, MC
is multiple choice. Full documentation for each question type and its format
can be found here:

https://www.csustan.edu/sites/default/files/blackboard/FacultyHelp/Documents/UploadingQuestions.pdf

>

Reply:
That doesn't explain what they *mean*. It just says how to format them in the csv file

Reply:
I think its more clear now. Theres a link for what the questions are, and the link for what each question's code is e.g. MC